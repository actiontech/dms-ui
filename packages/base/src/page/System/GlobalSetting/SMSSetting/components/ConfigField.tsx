import { useTranslation } from 'react-i18next';

import { FormItemLabel } from '@actiontech/shared/lib/components/CustomForm';
import { BasicInput } from '@actiontech/shared';

const ConfigField = () => {
  const { t } = useTranslation();

  return (
    <>
      <FormItemLabel
        className="has-required-style"
        label="url"
        name="url"
        rules={[{ required: true }]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: 'url'
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label="token"
        name="token"
        rules={[{ required: true }]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: 'token'
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
