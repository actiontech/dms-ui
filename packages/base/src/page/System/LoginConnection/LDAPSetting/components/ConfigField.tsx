import { useTranslation } from 'react-i18next';

import { BasicInput, BasicSwitch } from '@actiontech/shared';
import {
  CustomLabelContent,
  FormItemLabel
} from '@actiontech/shared/lib/components/CustomForm';

import { validatorPort } from '@actiontech/shared/lib/utils/FormRule';

const ConfigField = () => {
  const { t } = useTranslation();
  return (
    <>
      <FormItemLabel
        label={t('dmsSystem.ldap.enableLdapSSL')}
        name="enable_ssl"
        valuePropName="checked"
      >
        <BasicSwitch />
      </FormItemLabel>
      <FormItemLabel
        label={t('dmsSystem.ldap.ldapServerHost')}
        name="ldap_server_host"
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapServerHost')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        label={t('dmsSystem.ldap.ldapServerPort')}
        name="ldap_server_port"
        rules={[
          {
            validator: validatorPort()
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapServerPort')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-label-tip"
        label={
          <CustomLabelContent
            title={t('dmsSystem.ldap.ldapConnectDn')}
            tips={t('dmsSystem.ldap.ldapConnectDnTips')}
          />
        }
        name="ldap_connect_dn"
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapConnectDn')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        label={t('dmsSystem.ldap.ldapConnectPwd')}
        name="ldap_connect_pwd"
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapConnectPwd')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-label-tip"
        label={
          <CustomLabelContent
            title={t('dmsSystem.ldap.ldapSearchBaseDn')}
            tips={t('dmsSystem.ldap.ldapSearchBaseDnTips')}
          />
        }
        name="ldap_search_base_dn"
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapSearchBaseDn')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-label-tip"
        label={
          <CustomLabelContent
            title={t('dmsSystem.ldap.ldapUserNameRdnKey')}
            tips={t('dmsSystem.ldap.ldapUserNameRdnKeyTips')}
          />
        }
        name="ldap_user_name_rdn_key"
      >
        <BasicInput placeholder={t('common.form.placeholder.input')} />
      </FormItemLabel>
      <FormItemLabel
        className="has-label-tip"
        label={
          <CustomLabelContent
            title={t('dmsSystem.ldap.ldapUserEmailRdnKey')}
            tips={t('dmsSystem.ldap.ldapUserEmailRdnKeyTips')}
          />
        }
        name="ldap_user_email_rdn_key"
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.ldap.ldapUserEmailRdnKey')
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
