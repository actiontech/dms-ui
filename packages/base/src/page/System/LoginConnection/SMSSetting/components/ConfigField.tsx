import { useTranslation } from 'react-i18next';

import {
  CustomLabelContent,
  FormItemLabel
} from '@actiontech/shared/lib/components/CustomForm';
import { BasicInput } from '@actiontech/shared';

const ConfigField = () => {
  const { t } = useTranslation();

  return (
    <>
      <FormItemLabel
        className="has-required-style has-label-tip"
        label={
          <CustomLabelContent
            title="url"
            tips={t('dmsSystem.global.smsSetting.urlTips')}
          />
        }
        name="url"
        rules={[{ required: true }]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: 'url'
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style has-label-tip"
        label={
          <CustomLabelContent
            title="token"
            tips={t('dmsSystem.global.smsSetting.tokenTips')}
          />
        }
        name="token"
        rules={[{ required: true }]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: 'token'
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
