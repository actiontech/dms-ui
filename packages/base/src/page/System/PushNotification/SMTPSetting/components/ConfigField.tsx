import { useTranslation } from 'react-i18next';
import { BasicInput, BasicSwitch } from '@actiontech/shared';
import {
  CustomLabelContent,
  FormItemLabel
} from '@actiontech/shared/lib/components/CustomForm';
import { validatorPort } from '@actiontech/shared/lib/utils/FormRule';

const ConfigField = () => {
  const { t } = useTranslation();

  return (
    <>
      <FormItemLabel
        className="has-label-tip"
        label={
          <CustomLabelContent
            title={t('dmsSystem.smtp.isSkipVerify')}
            tips={t('dmsSystem.smtp.skipVerifyTips')}
          />
        }
        name="isSkipVerify"
        valuePropName="checked"
      >
        <BasicSwitch />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.smtp.host')}
        name="host"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.smtp.host')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.smtp.port')}
        name="port"
        rules={[
          {
            required: true
          },
          {
            validator: validatorPort()
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.smtp.port')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.smtp.username')}
        name="username"
        rules={[
          {
            required: true
          },
          {
            type: 'email'
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.smtp.username')
          })}
        />
      </FormItemLabel>

      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.smtp.password')}
        name="password"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.smtp.password')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.smtp.passwordConfirm')}
        name="passwordConfirm"
        dependencies={['password']}
        rules={[
          {
            required: true
          },
          ({ getFieldValue }) => ({
            validator(_, value) {
              if (!value || getFieldValue('password') === value) {
                return Promise.resolve();
              }
              return Promise.reject(
                new Error(t('common.form.rule.passwordNotMatch'))
              );
            }
          })
        ]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.smtp.passwordConfirm')
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
