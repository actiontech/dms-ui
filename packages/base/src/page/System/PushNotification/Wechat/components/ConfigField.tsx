import { useTranslation } from 'react-i18next';
import { BasicInput, BasicSwitch } from '@actiontech/dms-kit';
import { FormItemLabel } from '@actiontech/dms-kit';
const ConfigField = () => {
  const { t } = useTranslation();
  return (
    <>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.wechat.corp_id')}
        name="corp_id"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.wechat.corp_id')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.wechat.agent_id')}
        name="agent_id"
        rules={[
          {
            required: true
          },
          {
            pattern: /^\d*$/
          }
        ]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.wechat.agent_id')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.wechat.corp_secret')}
        name="corp_secret"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.wechat.corp_secret')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        label={t('dmsSystem.wechat.safe_enabled')}
        name="safe_enabled"
        valuePropName="checked"
      >
        <BasicSwitch />
      </FormItemLabel>
      <FormItemLabel label={t('dmsSystem.wechat.proxy_ip')} name="proxy_ip">
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.wechat.proxy_ip')
          })}
        />
      </FormItemLabel>
    </>
  );
};
export default ConfigField;
