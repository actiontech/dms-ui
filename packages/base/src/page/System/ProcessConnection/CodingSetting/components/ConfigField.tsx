import { useTranslation } from 'react-i18next';

import { FormItemLabel } from '@actiontech/shared/lib/components/CustomForm';
import { BasicInput } from '@actiontech/shared';

const ConfigField = () => {
  const { t } = useTranslation();

  return (
    <>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.codingDocking.serviceAddress')}
        name="codingUrl"
        rules={[{ required: true }]}
      >
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.codingDocking.serviceAddress')
          })}
        />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.codingDocking.accessToken')}
        name="token"
        rules={[{ required: true }]}
      >
        <BasicInput.Password
          placeholder={t('common.form.placeholder.input', {
            name: t('dmsSystem.codingDocking.accessToken')
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
