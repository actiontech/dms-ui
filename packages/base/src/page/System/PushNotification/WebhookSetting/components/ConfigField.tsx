import { useTranslation } from 'react-i18next';

import { BasicInput, BasicInputNumber } from '@actiontech/shared';
import { FormItemLabel } from '@actiontech/shared/lib/components/FormCom';

const ConfigField = () => {
  const { t } = useTranslation();

  return (
    <>
      <FormItemLabel
        className="has-required-style"
        label="Webhook url"
        name="url"
        rules={[
          {
            required: true,
            type: 'url'
          }
        ]}
      >
        <BasicInput />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.webhook.maxRetryTimes')}
        name="maxRetryTimes"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInputNumber min={0} max={5} />
      </FormItemLabel>
      <FormItemLabel
        className="has-required-style"
        label={t('dmsSystem.webhook.retryIntervalSeconds')}
        name="retryIntervalSeconds"
        rules={[
          {
            required: true
          }
        ]}
      >
        <BasicInputNumber min={1} max={5} />
      </FormItemLabel>
      <FormItemLabel label="token" name="token">
        <BasicInput
          placeholder={t('common.form.placeholder.input', {
            name: 'token'
          })}
        />
      </FormItemLabel>
    </>
  );
};

export default ConfigField;
