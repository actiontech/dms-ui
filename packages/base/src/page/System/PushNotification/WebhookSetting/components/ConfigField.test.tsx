import { renderWithTheme } from '@actiontech/shared/lib/testUtil/customRender';
import { act, cleanup, fireEvent } from '@testing-library/react';
import { getBySelector } from '@actiontech/shared/lib/testUtil/customQuery';

import { Form } from 'antd';
import ConfigField from './ConfigField';

describe('base/System/PushNotification/WebhookSetting/ConfigField', () => {
  const customRender = () => {
    return renderWithTheme(
      <Form>
        <ConfigField />
      </Form>
    );
  };

  beforeEach(() => {
    jest.useFakeTimers();
  });

  afterEach(() => {
    jest.useRealTimers();
    jest.clearAllMocks();
    cleanup();
  });

  it('render snap', () => {
    const { baseElement } = customRender();
    expect(baseElement).toMatchSnapshot();
  });

  it('render validate url', async () => {
    const { baseElement } = customRender();

    fireEvent.change(getBySelector('#url', baseElement));
    await act(async () => jest.advanceTimersByTime(500));
    expect(baseElement).toMatchSnapshot();
  });
});
