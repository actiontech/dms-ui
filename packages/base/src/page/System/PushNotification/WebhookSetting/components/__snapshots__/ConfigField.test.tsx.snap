// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`base/System/PushNotification/WebhookSetting/ConfigField render snap 1`] = `
<body>
  <div>
    <form
      class="ant-form ant-form-horizontal css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="url"
              title="Webhook url"
            >
              Webhook url
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <input
                  aria-required="true"
                  class="ant-input css-dev-only-do-not-override-ywkr0o basic-input-wrapper css-1pd1cd5"
                  id="url"
                  placeholder="请输入{{name}}"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="maxRetryTimes"
              title="最大重试次数"
            >
              最大重试次数
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-input-number ant-input-number-lg ant-input-number-in-form-item css-dev-only-do-not-override-ezmdel basic-inputNumber-wrapper css-cr6b1y"
                >
                  <div
                    class="ant-input-number-handler-wrap"
                  >
                    <span
                      aria-disabled="false"
                      aria-label="Increase Value"
                      class="ant-input-number-handler ant-input-number-handler-up"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="up"
                        class="anticon anticon-up ant-input-number-handler-up-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-disabled="false"
                      aria-label="Decrease Value"
                      class="ant-input-number-handler ant-input-number-handler-down"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-input-number-handler-down-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                  <div
                    class="ant-input-number-input-wrap"
                  >
                    <input
                      aria-required="true"
                      aria-valuemax="5"
                      aria-valuemin="0"
                      autocomplete="off"
                      class="ant-input-number-input"
                      id="maxRetryTimes"
                      role="spinbutton"
                      step="1"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="retryIntervalSeconds"
              title="最大重试间隔(秒)"
            >
              最大重试间隔(秒)
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-input-number ant-input-number-lg ant-input-number-in-form-item css-dev-only-do-not-override-ezmdel basic-inputNumber-wrapper css-cr6b1y"
                >
                  <div
                    class="ant-input-number-handler-wrap"
                  >
                    <span
                      aria-disabled="false"
                      aria-label="Increase Value"
                      class="ant-input-number-handler ant-input-number-handler-up"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="up"
                        class="anticon anticon-up ant-input-number-handler-up-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-disabled="false"
                      aria-label="Decrease Value"
                      class="ant-input-number-handler ant-input-number-handler-down"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-input-number-handler-down-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                  <div
                    class="ant-input-number-input-wrap"
                  >
                    <input
                      aria-required="true"
                      aria-valuemax="5"
                      aria-valuemin="1"
                      autocomplete="off"
                      class="ant-input-number-input"
                      id="retryIntervalSeconds"
                      role="spinbutton"
                      step="1"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class=""
              for="token"
              title="token"
            >
              token
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <input
                  class="ant-input css-dev-only-do-not-override-ywkr0o basic-input-wrapper css-1pd1cd5"
                  id="token"
                  placeholder="请输入token"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;

exports[`base/System/PushNotification/WebhookSetting/ConfigField render validate url 1`] = `
<body>
  <div>
    <form
      class="ant-form ant-form-horizontal css-dev-only-do-not-override-txh9fw"
    >
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="url"
              title="Webhook url"
            >
              Webhook url
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <input
                  aria-required="true"
                  class="ant-input css-dev-only-do-not-override-ywkr0o basic-input-wrapper css-1pd1cd5"
                  id="url"
                  placeholder="请输入{{name}}"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="maxRetryTimes"
              title="最大重试次数"
            >
              最大重试次数
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-input-number ant-input-number-lg ant-input-number-in-form-item css-dev-only-do-not-override-ezmdel basic-inputNumber-wrapper css-cr6b1y"
                >
                  <div
                    class="ant-input-number-handler-wrap"
                  >
                    <span
                      aria-disabled="false"
                      aria-label="Increase Value"
                      class="ant-input-number-handler ant-input-number-handler-up"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="up"
                        class="anticon anticon-up ant-input-number-handler-up-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-disabled="false"
                      aria-label="Decrease Value"
                      class="ant-input-number-handler ant-input-number-handler-down"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-input-number-handler-down-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                  <div
                    class="ant-input-number-input-wrap"
                  >
                    <input
                      aria-required="true"
                      aria-valuemax="5"
                      aria-valuemin="0"
                      autocomplete="off"
                      class="ant-input-number-input"
                      id="maxRetryTimes"
                      role="spinbutton"
                      step="1"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item has-required-style css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class="ant-form-item-required"
              for="retryIntervalSeconds"
              title="最大重试间隔(秒)"
            >
              最大重试间隔(秒)
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <div
                  class="ant-input-number ant-input-number-lg ant-input-number-in-form-item css-dev-only-do-not-override-ezmdel basic-inputNumber-wrapper css-cr6b1y"
                >
                  <div
                    class="ant-input-number-handler-wrap"
                  >
                    <span
                      aria-disabled="false"
                      aria-label="Increase Value"
                      class="ant-input-number-handler ant-input-number-handler-up"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="up"
                        class="anticon anticon-up ant-input-number-handler-up-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="up"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M890.5 755.3L537.9 269.2c-12.8-17.6-39-17.6-51.7 0L133.5 755.3A8 8 0 00140 768h75c5.1 0 9.9-2.5 12.9-6.6L512 369.8l284.1 391.6c3 4.1 7.8 6.6 12.9 6.6h75c6.5 0 10.3-7.4 6.5-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                    <span
                      aria-disabled="false"
                      aria-label="Decrease Value"
                      class="ant-input-number-handler ant-input-number-handler-down"
                      role="button"
                      unselectable="on"
                    >
                      <span
                        aria-label="down"
                        class="anticon anticon-down ant-input-number-handler-down-inner"
                        role="img"
                      >
                        <svg
                          aria-hidden="true"
                          data-icon="down"
                          fill="currentColor"
                          focusable="false"
                          height="1em"
                          viewBox="64 64 896 896"
                          width="1em"
                        >
                          <path
                            d="M884 256h-75c-5.1 0-9.9 2.5-12.9 6.6L512 654.2 227.9 262.6c-3-4.1-7.8-6.6-12.9-6.6h-75c-6.5 0-10.3 7.4-6.5 12.7l352.6 486.1c12.8 17.6 39 17.6 51.7 0l352.6-486.1c3.9-5.3.1-12.7-6.4-12.7z"
                          />
                        </svg>
                      </span>
                    </span>
                  </div>
                  <div
                    class="ant-input-number-input-wrap"
                  >
                    <input
                      aria-required="true"
                      aria-valuemax="5"
                      aria-valuemin="1"
                      autocomplete="off"
                      class="ant-input-number-input"
                      id="retryIntervalSeconds"
                      role="spinbutton"
                      step="1"
                      value=""
                    />
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
      </div>
      <div
        class="ant-form-item css-1jlm9cy css-dev-only-do-not-override-txh9fw"
      >
        <div
          class="ant-row ant-form-item-row css-dev-only-do-not-override-txh9fw"
        >
          <div
            class="ant-col ant-form-item-label css-dev-only-do-not-override-txh9fw"
          >
            <label
              class=""
              for="token"
              title="token"
            >
              token
            </label>
          </div>
          <div
            class="ant-col ant-form-item-control css-dev-only-do-not-override-txh9fw"
          >
            <div
              class="ant-form-item-control-input"
            >
              <div
                class="ant-form-item-control-input-content"
              >
                <input
                  class="ant-input css-dev-only-do-not-override-ywkr0o basic-input-wrapper css-1pd1cd5"
                  id="token"
                  placeholder="请输入token"
                  type="text"
                  value=""
                />
              </div>
            </div>
          </div>
        </div>
      </div>
    </form>
  </div>
</body>
`;
